src/cdk/utils/create-lambda-integration.ts

```diff
@@ -10,15 +10,11 @@ import * as path from 'path';
 import {LambdaConfig} from '../../types';
 import {createShortHash} from '../../utils/create-short-hash';
 
-const noneAuthorizer: IAuthorizer = {
-  authorizerId: 'none',
-  authorizationType: AuthorizationType.NONE
-};
-
 export function createLambdaIntegration(
   stack: Stack,
   restApi: RestApi,
-  lambdaConfig: LambdaConfig
+  lambdaConfig: LambdaConfig,
+  authorizer?: IAuthorizer
```

Comment:
I thin this should be a non-optional parameter.

Reply:
Agree